eqcorrscan/utils/plotting.py

```diff
@@ -1246,17 +1248,32 @@ def noise_plot(signal, noise, normalise=False, **kwargs):
     plt.figlegend(lines, labels, 'upper left')
     plt.tight_layout()
     plt.subplots_adjust(hspace=0)
-    fig = _finalise_figure(fig=fig, **kwargs)  # pragma: no cover
-    return fig
+    plt.subplots_adjust(top=0.91)
+    if plotdir:
```

Comment:
This logic should be in the calling function, see earlier comment.  savefile, title, etc should be handled by kwargs.